src/domain/flow-system/services/src/flow/flow_executor_impl.rs

```diff
@@ -57,13 +57,13 @@ pub struct FlowExecutorImpl {
 #[interface(dyn MessageConsumer)]
 #[interface(dyn MessageConsumerT<TaskProgressMessage>)]
 #[interface(dyn MessageConsumerT<DatasetLifecycleMessage>)]
-#[interface(dyn MessageConsumerT<FlowConfigurationUpdatedMessage>)]
+#[interface(dyn MessageConsumerT<FlowTriggerUpdatedMessage>)]
```

Comment:
Does this event really need to be processed transactionally? Isn't it only affects in-memory state?

Reply:
It can launch new flows, this requires a transaction.